docs/source/usage/add-diagnostic.rst

```diff
@@ -102,7 +102,7 @@ on the command line with the `cset bake` command and without running the whole
 workflow. This is a useful step for testing new operators or recipes during
 development.
 
-Now we are linking the ``src/CSET/recipes/<new_recipe_name.yml>``.
+Now we are linking the ``src/CSET/recipes/<new_recipe_name.yml>`` into the workflow.
```

Comment:
```suggestion
Now we are linking the ``src/CSET/recipes/<new_recipe_name.yml>`` into
the cset command line program.
```
This section adds the recipe into the command line (`cset bake`, `cset cookbook`); it is the following section that adds it to the workflow.